OrbitLinuxTracing/PerfEventReaders.cpp

```diff
@@ -0,0 +1,15 @@
+#include "PerfEventReaders.h"
+
+#include <OrbitBase/Logging.h>
+#include "PerfEventRingBuffer.h"
+
+namespace LinuxTracing {
+
+uint16_t ReadTracepointCommonType(PerfEventRingBuffer* ring_buffer) {
```

Comment:
I can see that you like functions being defined in a cpp file, but is it that necessary to do differently than all the other functions here? It makes this one look special. It's even a good candidate for inlining...

Reply:
I actually think we should move them all to a cpp file. There's no reason for inlining them and it's easier to debug if they are defined in a cpp file. I can take care of moving them.

Reply:
Moved all functions to the cpp file.

Reply:
Alright then.

Reply:
I did a quick benchmark using "perf record" on OrbitService while instrumenting the top ~20 functions and none of the read functions here show up in the profile at all. Do you have any data that indicates these should be inlined?